install_config/configuring_local.adoc

```diff
@@ -278,11 +294,11 @@ $ ln -s /dev/dm-36 dm-uuid-LVM-1234
 +
 [NOTE]
 ====
-It is a good practice to use the same name for the symbolic link as the link
-from *_/dev/disk/by-uuid_* or *_/dev/disk/by-id_* directory to avoid possible name conflicts.
+To avoid possible name conflicts, it is recommended to use the same name for the symbolic link and the link
```

Comment:
s/it is recommended to use/use
(I know this language is in the glossary, but it's still not necessary. Brice called me out on this last week.)